.github/workflows/update-manpage.yml

```diff
@@ -0,0 +1,76 @@
+name: Update manpage and completions
+
+on:
+  push:
+    paths:
+      - .github/workflows/update-manpage.yml
+    branches-ignore:
+      - master
```

Comment:
I'm still confused with this, a little. Why are we ignoring pushes to `master`? If not running on merges to this workflow file, when will the `push` event be triggered?

Reply:
That's actually the same thing I was confused about (as I'd asked above in the other comment thread) - however this workflow runs on pull requests that modify these files, and so that's when the PR would be created. I'm unsure if this is what Mike meant., though and it's possible I've misunderstood what was said.

Reply:
Hmm, it looks like its running when a pull request is opened. I don't think that's what's desired, here, though. If I create a PR, I don't want a follow-up PR fixing the manpages opened until _after_ my PR is merged, right?

> As for the `push` event, that would be triggered when we make changes to the workflow file itself on a branch of Homebrew/brew, and want to test the changes.

Okay, that makes sense. However, we should _also_ have this run on `master`, right?

Reply:
> Okay, that makes sense. However, we should _also_ have this run on `master`, right?

I think the `master` check should be scheduled instead of on `push`.



> That's actually the same thing I was confused about (as I'd asked above in the other comment thread) - however this workflow runs on pull requests that modify these files, and so that's when the PR would be created (`pull_request`). I'm unsure if this is what Mike meant, and it's possible I've misunderstood what was said.

Apologies for the confusion. I was thinking this could, like the vendor gems workflow, commit the manpage changes directly to the (open, unmarked) PR. Thinking about this more, `brew style --fix` might be another good candidate for that.

Reply:
> If I create a PR, I don't want a follow-up PR fixing the manpages opened until after my PR is merged, right?

Yes, that's what I thought initially, I guess I got confused along the way. Let me revise this and we can address feedback in another iteration of the review.

Reply:
Ah, sorry Mike, I seem to have missed your comment.

> Apologies for the confusion. I was thinking this could, like the vendor gems workflow, commit the manpage changes directly to the (open, unmarked) PR.

I thought committing to the PR branch would be a problem in case things change upon code review, especially with things like options.

Reply:
> I thought committing to the PR branch would be a problem in case things change upon code review, especially with things like options.

Gotcha. In that case, yeh, I'd make it scheduled and on pushes to `master` only. Sorry for the confusion!

Reply:
No problem at all Mike, glad to get things cleared up 🙂